### DISCUSSION TITLE
no ssr rendering on netlify with ssr=true

### DISCUSSION THREAD

**Discussion by itsmnthn** [nuxt/nuxt#17191] [Questions]
With the latest update I am not able to get SSR working with Nuxt3 deploying on netlify I just update the from rc.4 to rc.6 and it now don't server render meta tags and page

Before
<img width="1092" alt="image" src="https://user-images.githubusercontent.com/25129674/181263681-33a255d5-8bbc-4ed6-b587-16fbd1dc59c1.png">

After: Current Page source
<img width="813" alt="image" src="https://user-images.githubusercontent.com/25129674/181262027-1d9ac213-2ffc-40e7-919b-87c96dcb3fb5.png">

Same page with no config change or anything

**Comment by itsmnthn** [comment]
Also I'm getting `javascript heap out of memory` with nothing changed

**Reply by alirezaaghaeee** [reply]
i have same problem 
did you fined any new thing?
